scripting/storing/manipulation-object-detector.sh

```diff
@@ -0,0 +1,6 @@
+SESSION_NAME="manipulation-receptionist-arm"
+
+tmux new-session -d -s $SESSION_NAME
+tmux send-keys -t $SESSION_NAME 'cd ../../' C-m
+tmux send-keys -t $SESSION_NAME 'bash run.sh manipulation' C-m
+tmux send-keys -t $SESSION_NAME 'ros2 launch object_detector_2d object_detector_node.launch.py yolo_model_path:=src/home2/vision/packages/object_detector_2d/models/yolo11classes.pt'
```

Comment:
Now the full model path will not be needed

```suggestion
tmux send-keys -t $SESSION_NAME 'ros2 launch object_detector_2d object_detector_node.launch.py yolo_model_path:=yolo11classes.pt'
```

Reply:
Also, the detector will now be included in vision launches